frontend/src/lib/apolloClient.ts

```diff
@@ -19,7 +19,7 @@ export type PageProps = {
 
 let apolloClient: ApolloClient<NormalizedCacheObject> | undefined;
 
-function createApolloClient(ctx?: GetServerSidePropsContext): ApolloClient<NormalizedCacheObject> {
+const createApolloClient = (ctx?: GetServerSidePropsContext): ApolloClient<NormalizedCacheObject> => {
```

Comment:
These have been deliberately declared using `function` as I believe it is clearer to have a distinction between regular variables and functions. Arrow functions are nice, they make sense in combination with React.FC, but for regular old javascript code I think we should stick to functions. 

While there are more differences between them than the stylistic ones, none of them come into play here.

Reply:
Fair enough, changed back in 9f2d160.